cmd/bucket-replication.go

```diff
@@ -1916,6 +1933,12 @@ func getLatestReplicationStats(bucket string, u BucketUsageInfo) (s BucketReplic
 	return s
 }
 
+// get the most current of in-memory replication stats  and data usage info from crawler.
+func getLatestReplicationStats(bucket string, u BucketUsageInfo) (s BucketReplicationStats) {
+	bucketStats := globalNotificationSys.GetClusterBucketStats(GlobalContext, bucket)
```

Comment:
I would say let's add a context with a timeout at metrics-v2.go level. Though it is not very clear in my mind if this is really needed, we have a dial timeout of 10 seconds for internode connections but maybe we need to a broader timeout for all the network call steps here